generated-antlr/build.gradle.kts

```diff
@@ -32,3 +35,26 @@ dependencies {
 tasks.named<Jar>("sourcesJar") {
     dependsOn(tasks.named("generateGrammarSource"))
 }
+
+tasks.named<Jar>("jar") {
+    archiveClassifier.set("raw")
+}
+
+tasks.named<ShadowJar>("shadowJar") {
+    // The antlr-plugin should ideally do this
+    dependsOn(tasks.named("generateGrammarSource"))
+
+    dependencies {
+        include(dependency("org.antlr:antlr4-runtime"))
+    }
+    relocate("org.antlr.v4.runtime", "org.projectnessie.cel.parser.gen.antlr.v4.runtime")
```

Comment:
Does it make sense if we keep the shaded package name as it is and add it `shaded`? e.g: `org.projectnessie.cel.shaded.org.antlr.v4.runtime`. In order to distinguish that is shaded from `org.antlr.v4.runtime`

Reply:
Yup - updated the package name